.github/workflows/e2e-tests-preview.yml

```
@@ -0,0 +1,465 @@
+            mobile-*)
+              echo "📱 Mobile browser debug information:"
+              echo "  Using device emulation for mobile testing"
+              ;;
+          esac
+          
+          # Check if there were any browser-specific timeouts
+          if [ -f "test-results" ]; then
+            echo "  Timeout errors:"
+            find test-results -name "*.txt" -exec grep -l "timeout\|TimeoutError" {} \; 2>/dev/null || echo "    No timeout errors found"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)